chapters/object_types/spatial_fields.txt

```diff
@@ -55,6 +55,39 @@ their addresses in memory can be easily computed from a 3D position. A
 common type of structured spatial fields are regular grids.
 ====
 
+[[object_types_spatial_field_nanovdb]]
+==== NanoVDB
+
+Extension `KHR_SPATIAL_FIELD_NANOVDB`
+
+NanoVDB spatial fields are created by passing the subtype string
+`nanovdb` to <<object_types_spatial_field, `anariNewSpatialField`>>.
+The parameters understood by NanoVDB spatial fields are summarized in
+the table below.
+
+.Configuration parameters for NanoVDB spatial fields.
+[cols="<,<2,>,<5",options="header,unbreakable"]
+|===================================================================================================
+| Name    | Type                |    Default | Description
+| data    |`ARRAY1D` of `UINT8` |            | NanoVDB grid, as a binary blob.
+| filter  |`STRING`             |    `linear`| filter used for reconstructing the field,
+                                               possible values: `nearest`, `linear`, `cubic`
+|===================================================================================================
+
+The `data` parameter content is expected to be a NanoVDB grid, as read from a NanoVDB file or constructed by the NanoVDB API.
+The spatial field extent is implicitely defined in the grid header.
+
+Setting `data` to an incompatible or invalid NanoVDB grid should result in the device implementation emitting a warning and ignoring the invalid object.
```

Comment:
Potential incompatibility b/c of versions is troublesome. 
https://github.com/AcademySoftwareFoundation/openvdb/blob/master/nanovdb/nanovdb/NanoVDB.h#L146 says major version is linked to the (opaque) byte stream, which fortunately has not changed in 4 years. Should we make this more explicit, e.g., it's a `NanoVDBv32` spatial field? If the format changes, we can add a `NanoVDBv33` extension, and apps can thus see which to take (and which is supported). At least we should state the major version number, and add a `NanoVDB2` for version 33 in future (if at all).

Reply:
I've been discussing this with @jeffamstutz. We came to the conclusion that it might be better to:
- NOT tie the extension to a specific version of NanoVDB
- Add a new device property to query the version supported by the device implementing NanoVDB support.

How does that sound to you?

Reply:
A property query requires apps to create a device first. Yet, the version information ist static (it depends only on the used NanoVDB headers at time of compilation, doesn't it?), facilitating `anariGetDeviceExtensions`.